### DISCUSSION TITLE
Umzug der  Phoniebox

### DISCUSSION THREAD

**Discussion by Rei85An** [MiczFlor/RPi-Jukebox-RFID#1068] [Q&A] [answered]
Nach längerem bauen funktioniert nun meine Phoniebox. Zwar mit 1-2 Bugs aber mit denen kann ich aktuelle leben. (Werde ich demnächst nochmals erläutern)

Was nun viel wichtiger für mich wäre ist das ich die Box nun in ein neues WLAN integrieren möchte. Am besten ohne viel Aufwand! Ich habe leider keinen Bildschirm oder Tastatur auf die ich zurückgreifen kann.

Gibt es dafür eine Lösung ohne das ich alles komplett neu aufsetzt muss?

Vielen Dank und Gruß von einem blutigem Anfänger der es immerhin bis hierher geschafft hat 👍

**Comment by Freacly** [comment]
Über das Webinterface kannst du dies problemlos realisieren.

**Comment by Rei85An** [comment]
Das Problem ist das ich nun im neuen WLAN sitze, der Pi aber noch ein anderes WLAN inne hat!

**Comment by Ardios** [comment]
@Rei85An 

Da gibt es verschiedene Möglichkeiten...

Du kannst eine Tastatur, Maus und Monitor am Raspberry anschließen und dann das WLAN wechseln oder du greifst mit deinem PC/Book auf die Linux Partition der SD Karte zu und änderst dann manuell deine WLAN Daten (SSID, PW etc.) in der wpa_supplicant.conf

Es gibt mit Sicherheit noch andere Wege...dies sind nur meine persönlichen Erfahrungen ;)

Grüße

**Comment by s-martin** [comment, marked as answer]
> und dann das WLAN wechseln oder du greifst mit deinem PC/Book auf die Linux Partition der SD Karte zu und änderst dann manuell deine WLAN Daten (SSID, PW etc.) in der wpa_supplicant.conf

So: https://raspberrypi.stackexchange.com/a/57023

**Reply by alkfhasdjkfhaksjdfhakjdfaksdjf** [reply]
Hi, ist zwar schon Jahre her, dass gefragt wurde, aber vielleicht bekomme ich eine Antwort:
Bin leider auch völlig unfähig in all things coding, aber habe es doch geschafft, eine phoniebox für meine Kinder zusammenzubauen. Wir sind umgezogen, ich wollte das wlan neu konfigurieren indem ich die SD Karte am Computer anschließe und wie in der verlinkten Anleitung die Daten ändere. Ich kann aber zumindest auf diese Art kein wpa_supplicant.conf file finden. Was mache ich falsch? (Bzw. welche Informationen fehlen, um mir helfen zu können =) )

**Reply by AlvinSchiller** [reply]
Hi,
wenn die Datei in der "boot" Partition nicht vorhanden ist, dann einfach neu anlegen. Der inhalt wird dann beim Start kopiert und die Datei wieder gelöscht.
Hier ist das auch nochmal im Wiki beschrieben, wie der Inhalt sein soll: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/CONFIGURE#installation-and-configuration-via-ssh--headless-installation
Wichtig ist noch dass die Datei Unix-LineEndings hat (nur LF). Das lässt sich im Notepad++ z.B. unten rechts in der Leiste auswählen.
